modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -1,270 +1,306 @@
+Method Label Get Background Color:: Specifies the *GET* method label background color.
+Method Label Get Color:: Specifies the *GET* method label color.
+Method Label Post Background Color:: Specifies the *POST* method label background color.
+Method Label Post Color:: Specifies the *POST* method label color.
+Method Label Put Background Color:: Specifies the *PUT* method label background color.
+Method Label Put Color:: Specifies the *PUT* method label color.
+Method Label Delete Background Color:: Specifies the *DELETE* method label background color.
+Method Label Delete Color:: Specifies the *DELETE* method label color.
+Method Label Patch Background Color:: Specifies the *PATCH* method label background color.
+Method Label Patch Color:: Specifies the *PATCH* method label color.
+Method Label Options Background Color:: Specifies the *OPTIONS* method label background color.
+Method Label Options Color:: Specifies the *OPTIONS* method label color.
+Method Label Head Background Color:: Specifies the *HEAD* method label background color.
+Method Label Head Color:: Specifies the *HEAD* method label color.
+Link Hover Color:: Specifies the color of links when the cursor hovers over them.
+Button Text Color (Optional):: Specifies the color of button text.
+Button Hover Text Color (Optional):: Specifies the color of button hover text. Default is the button text color.
+Headline Font Family (Optional):: Specifies the headline font family.
+Error Color (Optional):: Specifies the color of error messages.
 
 == API Details Button
 
-The API details button component provides portal users with details about an API.
+The *API Details Button* component provides portal users with details about an API.
+
+You can specify values for the following parameters of the *API Details Button*:
 
-API:: Choose the API to display.
-Button Label:: Set the button label. Default is `API Details`.
-Button Align:: Set button alignment to left, right, or center of the UI field.
-API Name:: Add the API name to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
-API Version:: Add the API version to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
+API:: Specifies the API to display.
+Button Label:: Specifies the button label. Default is `API Details`.
+Button Align:: Specifies button alignment as left, right, or center of the UI field.
+API Name:: Adds the API name to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
+API Version:: Adds the API version to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
 
 == API Documentation Viewer
 
-The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+The *API Documentation Viewer* component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
```

Comment:
Do not use "may." See style reference. GLOBAL

Reply:
Removed "may."